lib/standard_api/views/application/_record.json.jbuilder

```diff
@@ -6,39 +6,42 @@ end
 
 includes.each do |inc, subinc|
   next if ["limit", "offset", "order", "when", "where", "distinct", "distinct_on"].include?(inc)
-  
+
+
   case association = record.class.reflect_on_association(inc)
-  when ActiveRecord::Reflection::HasManyReflection, ActiveRecord::Reflection::HasAndBelongsToManyReflection, ActiveRecord::Reflection::ThroughReflection
-    can_cache = can_cache_relation?(record.class, inc, subinc)
-    json.cache_if!(can_cache, can_cache ? association_cache_key(record, inc, subinc) : nil) do
-      partial = model_partial(association.klass)
-      json.set! inc do
-        # TODO limit causes preloaded assocations to reload
-        sub_records = record.send(inc)
-        
-        sub_records = sub_records.limit(subinc['limit']) if subinc['limit']
-        sub_records = sub_records.offset(subinc['offset']) if subinc['offset']
-        sub_records = sub_records.order(subinc['order']) if subinc['order']
-        sub_records = sub_records.filter(subinc['where']) if subinc['where']
-        sub_records = sub_records.distinct if subinc['distinct']
-        sub_records = sub_records.distinct_on(subinc['distinct_on']) if subinc['distinct_on']
+  when ActiveRecord::Reflection::AbstractReflection
```

Comment:
Is anything not an AbstractReflection?

Reply:
Not everything that is passed in is a relation. You can also include things that are not relations, like methods on the model.